package.json

```diff
@@ -2,7 +2,7 @@
   "name": "postcss-nesting",
   "description": "Nest rules inside each other in CSS",
   "license": "CC0-1.0",
-  "version": "9.0.1",
+  "version": "10.0.0",
```

Comment:
I'd revert this change since `npm version major` will take care of this and the tagging along